apis/apps/v1alpha1/cluster_types.go

```diff
@@ -654,12 +654,12 @@ type ClusterComponentSpec struct {
 	// +optional
 	EnabledLogs []string `json:"enabledLogs,omitempty"`
 
-	// Specifies Labels to override or add for underlying Pods.
+	// Specifies Labels to override or add for underlying Pods, Account Secrets, ConfigMaps, Services Owned by Component.
```

Comment:
Does the services here refer to cluster and component services, or just component services? The cluster services can specify their own labels and selector.

Reply:
> Does the services here refer to cluster and component services, or just component services? The cluster services can specify their own labels and selector.

only for component services and does not include headless svc